### PR TITLE
Fixes #10384: Fix some details in Directives page

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://www.rudder-project.org/redmine/issues/10384

### CODE REVIEW

File: rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayDirectiveTree.scala [hunk 1]
```diff
@@ -228,12 +229,11 @@ object DisplayDirectiveTree extends Loggable {
       )
 
       override def body = {
-
         val editButton = {
           if (addEditLink && ! directive.isSystem) {
             val tooltipId = Helpers.nextFuncName
             <span class="treeActions">
-              <span  class="tooltipable treeAction noRight directiveDetails fa fa-pencil" tooltipid={tooltipId} title="" onclick={redirectToDirectiveLink(directive.id).toJsCmd}> </span>
+              <span class="tooltipable treeAction noRight directiveDetails fa fa-pencil" tooltipid={tooltipId} title="" onclick={redirectToDirectiveLink(directive.id).toJsCmd}> </span>
```
Comment (VinceMacBuche) [inline rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayDirectiveTree.scala]: I think we could have updated that tooltip too (even if it only appears in the Rule page)

Comment (VinceMacBuche) [inline rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayDirectiveTree.scala]: What do you think @RaphaelGauthier ?

Comment (VinceMacBuche) [inline rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayDirectiveTree.scala]: Maybe we dan do it in another PR

Comment (RaphaelGauthier) [inline rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayDirectiveTree.scala]: I forgot this one, I think we should do it here! I update this PR

**Comment by RaphaelGauthier** [comment]
Commit modified
